sphinx_gherkindoc/writer.py

```diff
@@ -179,15 +183,22 @@ def tags(tags: List[str], *parent_objs: behave.model_core.BasicStatement) -> Non
             f"Tagged: {tag_str.strip()}", line_breaks=2, indent_by=INDENT_DEPTH
         )
 
-    def steps(steps: List[behave.model.Step]) -> None:
+    def format_step(step, step_format):
```

Comment:
One _tiny_ request. Since this package is `mypy`-friendly, can we add typehints to this signature?